### PR TITLE
Chat system

### PR DESCRIPTION
**Pull request by bensku**

### CODE REVIEW

File: src/chat/handlers/effect.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+interface ChatSounds {
+  normal: string[];
+  question: string[];
+}
+
+const CHAT_SOUNDS: Record<string, ChatSounds> = {
+  default: {
+    normal: ['minecraft:entity.villager.yes', 'minecraft:entity.villager.no'],
```
[the remark this answers is not in the document]
Comment (Laetta) [reply src/chat/handlers/effect.ts:27]: Same thing with  other playSounds

[the remark this answers is not in the document]
Comment (bensku) [reply src/chat/handlers/effect.ts:27]: Sound enum's documentation warns that it provides zero API stability over Vanilla sound names. Additionally, looking up sounds from Minecraft wiki is much simpler when Vanilla names are used as-is.

File: src/chat/handlers/mention.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Player } from 'org.bukkit.entity';
+import { getChatOption } from '../options';
+import { LOCAL_PIPELINE } from '../pipeline';
+
+export const IsMention = (props: string) => props;
```
[the remark this answers is not in the document]
Comment (bensku) [reply src/chat/handlers/mention.ts:5]: Yeah, it is intentional - I got the idea from React's functional components. Not that I'm yet sure if it is a *good* idea, but it wasn't a typo.

File: src/chat/items.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * Speaker staff significantly increases local channel range.
+ */
+export const SpeakerStaff = new CustomItem({
+  id: 1,
+  type: Material.GOLDEN_HORSE_ARMOR,
+  name: 'Puhujankeppi',
+  modelId: 1,
+});
+
+registerCommand('speakerstaff', (sender) => {
```
[the remark this answers is not in the document]
Comment (bensku) [reply src/chat/items.ts:15]: Nice! I think that was not the case when originally I added speaker staff code.

**Review by Laetta — APPROVED** [review]

**Pull request merged**